src/test/resources/regressions/features/defunc/pred-construct-fail1.gobra

```diff
@@ -0,0 +1,31 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+type T struct {
+	x int
+}
+
+pred (rec *T) AccInv() {
+	acc(rec)
+}
+
+func f(p pred())
+
+func error1() {
+	v := &T{}
+	fold v.AccInv()
+	//:: ExpectedOutput(type_error)
+	f((*T).AccInv!<v!>) // Predicate expressions are not allowed in predicate constructors
```

Comment:
Isn't this the same as `f(v.AccInv!<!>)`? And isn't this allowed?

Reply:
@Felalolf told me that Predicate Expressions in the base of predicate constructors will not make sense after he implements his latest proposal for interfaces.I don't recall the exact reason for this but maybe Felix can shed a light on this.